packages/twenty-front/src/pages/settings/data-model/SettingsObjectDetail.tsx

```diff
@@ -92,7 +195,25 @@ export const SettingsObjectDetail = () => {
     });
 
   const shouldDisplayAddFieldButton = !activeObjectMetadataItem.isRemote;
-
+  const tableHeadings: TableHeading = [
+    { label: 'Name', sortKey: SortKeys.label },
+    {
+      label: activeObjectMetadataItem.isCustom ? 'Identifier' : 'Field type',
+      sortKey: SortKeys.fieldType,
+    },
+    {
+      label: 'Data type',
+      sortKey: SortKeys.dataType,
+    },
+  ];
+  const SortIcon = ({ sortKey }: { sortKey: SortKeys }) => {
```

Comment:
This should be move in the component that handles the column heading, it should be abstracted down.